index.bs

```diff
@@ -192,6 +192,7 @@ interface ImageCapture {
     readonly attribute MediaSettingsRange         imageHeight;
     readonly attribute MediaSettingsRange         imageWidth;
     readonly attribute FrozenArray&lt;FillLightMode> fillLightMode;
+    readonly attribute FrozenArray&lt;DOMString>     supportedImageFormats;
```

Comment:
I see what you mean, sorry I was so thick, thanks for the explanation. The other cases in the current Spec that follow this pattern of `FrozenArray<DOMString>` as capabilities and `DOMString` for setting/constraining have the same name in both cases, e.g. [`exposureMode`](https://w3c.github.io/mediacapture-image/#dom-mediatrackcapabilities-exposuremode) and [`exposureMode`](https://w3c.github.io/mediacapture-image/#dom-mediatracksettings-exposuremode), so I'd say that for the purpose of this PR is better to follow that idea, and just use the name `imageFormats` leaving to the dictionary to resolve the semantics.  

If this still looks confusing we could consider changing all of them at the same time, but take into account that Cr already shipped those.